user/conditions-v0.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Conditions (v0, deprecated)
+layout: en
+---
+
+> This page documents the behavior of conditions `v0` (beta), which is
```

Comment:
I think we can drop `(beta)` here, as it's already deprecated?

Reply:
done!